coretex/cli/utils.py

```diff
@@ -14,3 +34,56 @@ def arrowPrompt(choices: List[Any]) -> Any:
     ])
 
     return answers["option"]
+
+
+def checkIfExpired(expirationDate: datetime) -> bool:
```

Comment:
There is no need for this to be a separate function. Use `datetime.utcnow()` directly and compare that value with the token expiration dates.